### ISSUE TITLE
What about the unit test?

### ISSUE THREAD

**Issue by semicircle** [martyjs/marty-chat-example#7]
There's a test section in the Makefile, but no unit test code included in this example.
I'm wondering how to write unit test for Marty?

**Comment by semicircle** [comment]
Or, would you please advice me which test library is suitable for Marty ? 
Jest or something else ?

**Comment by semicircle** [comment]
Sorry , I've found an issue about unit test in main repo, I didn't search there. 
But, I think it would be nice to have unit tests with this chat example.

**Comment by jhollingworth** [comment]
Yeah, I've been meaning to do this for a while. Will try and find some time to add some
